src/tridiag.jl

```diff
@@ -610,6 +601,7 @@ LA.svd(A::SkewHermTridiagonal) = svd!(copyeigtype(A))
 
 @views function to_symtridiagonal(A::SkewHermTridiagonal{T}) where {T<:Complex}
     n = size(A, 1)
+    n == 1 && return SymTridiagonal(-A.dvim,real(T)[]), Diagonal(T[1])
```

Comment:
Why is this special case necessary?   Shouldn't the code below work when `n == 1`?